browser/src/components/training/TrainingInformation.vue

```diff
@@ -9,7 +9,10 @@
     >
       <div class="grid grid-cols-2 p-4 gap-8">
```

Comment:
you can add the v-field `:class="hasValidationData ? grid-cols-2 : grid-cols-1"` next to `class="grid p-4 gap-8"`